Extension/translations_auto_pr.js

```diff
@@ -15,9 +15,12 @@ let repoName = process.argv[3];
 let locProjectName = process.argv[4];
 let authUser = process.argv[5];
 let authToken = process.argv[6];
+let userFullName = process.argv[7];
+let userEmail = process.argv[8];
+let locRepoPath = process.argv[9];
 
 if (!repoOwner || !repoName || !locProjectName || !authUser || !authToken) {
```

Comment:
Also and check for the new vars `user_full_name` and `user_email` and `loc_repo_path`?